### PR TITLE
Added security code to new copy of main

### PR DESCRIPTION
**Pull request by Steve-Rozeboom**

### CODE REVIEW

**Comment by Steve-Rozeboom** [comment]
screencapture-localhost-3000-error-logging-index-html-2024-05-21-11_56_37

File: apps/common-capabilities/src/pages/details/index.tsx [hunk 1]
```diff
@@ -109,6 +119,74 @@ export default function Details({ app }: DetailsProps): JSX.Element {
...
+    function displayName(obj: any, key: string): string | undefined {
+      return obj[key]?.title;
+    }
+
+    return (
+      <>
+        <GoASpacer vSpacing="xl" />
+        {group.note != '' ? (
+          <>
+            <span>{group.note}</span>{' '} {/* <GoAtext ????={group.note} ></GoAtext> */}
```
Comment (smettu007) [inline apps/common-capabilities/src/pages/details/index.tsx:147]: use a `p` tag here

File: apps/common-capabilities/src/pages/details/index.tsx [hunk 2]
```diff
@@ -164,81 +242,88 @@ export default function Details({ app }: DetailsProps): JSX.Element {
...
-                    </a>
-                  ) : (
-                    ''
-                  );
-                })
-              : 'No content'}
-          </GoASideMenu>
+    <>
+      <GoAThreeColumnLayout
+        nav={
```
Comment (smettu007) [inline apps/common-capabilities/src/pages/details/index.tsx:273]: can you add `maxContentWidth="1500px"` above `nav`

**Review by smettu007 — CHANGES_REQUESTED** [review]

File: apps/common-capabilities/src/pages/details/index.tsx (hunk 2 above)
Comment (Steve-Rozeboom) [reply apps/common-capabilities/src/pages/details/index.tsx:274]: done

**Review by smettu007 — APPROVED** [review]

**Pull request merged**